File: mobile/src/main/java/com/windscribe/mobile/ticket/SendTicketView.java
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.mobile.ticket;

public interface SendTicketView {

    void addTextChangeListener();

    void setActivityTitle(String title);

    void setEmail(String email);

    void setErrorLayout(String message);

    void setProgressView(boolean show);

    void setQueryTypeSpinner();

    void setSendButtonState(boolean enabled);

    void setSuccessLayout(String message);
}

File: mobile/src/main/java/com/windscribe/mobile/ticket/SendTicketView.kt
/*
 * Copyright (c) 2021 Windscribe Limited.
 */
package com.windscribe.mobile.ticket

interface SendTicketView {
    fun addTextChangeListener()
    fun setActivityTitle(title: String)
    fun setEmail(email: String)
    fun setErrorLayout(message: String)
    fun setProgressView(show: Boolean)
    fun setQueryTypeSpinner()
    fun setSendButtonState(enabled: Boolean)
    fun setSuccessLayout(message: String)
}